src/typography/typography.css

```diff
@@ -103,6 +103,10 @@ body {
   color: var(--omrs-color-brand-black);
 }
 
+.weirdClass {
```

Comment:
Can we name this better? `weirdClass` isn't very descriptive that this css class is for a font. Maybe something like `omrs-secondary-font`? Also, css classes in the styleguide use the traditional `-` syntax instead of camelCase. CSS Modules suggest camel case, but we're not using CSS modules here and should follow the common pattern for global css.

Reply:
Lol! 😆 I totally forgot to delete this, updating the PR in a few. Good catch!

Reply:
updated the PR @joeldenning.